### DISCUSSION TITLE
Finding orientation accuracy of Single crystal

### DISCUSSION THREAD

**Discussion by qmrphd** [mtex-toolbox/mtex#528] [Ask Anything]
I have 001 orientation Fe single crystal which seems few degrees out from exact 001 orientation. I want to find how Degree's its off from 001 orientation. Could you please let me know how it can be done using MTEX.

Kind Regards 

Qamar

**Comment by ralfHielscher** [comment]
Hi Qamar,

a few questions:

1. what is the 001 orientation?
2. what are your data, or in what form do you know the single crystal orientation - in Euler angles?

Ralf.
